src/main/java/frc/robot/Controls.java

```diff
@@ -642,20 +644,38 @@ private void configureElevatorLEDBindings() {
       Commands.waitSeconds(1)
           .andThen(
               s.elevatorLEDSubsystem
-                  .colorSet(255, 0, 0, "Red - Elevator Not Zeroed")
+                  .blink(120, 0, 0, "Red - Elevator Not Zeroed")
                   .ignoringDisable(true))
           .schedule();
       hasBeenZeroed.onTrue(
           s.elevatorLEDSubsystem
-              .colorSet(0, 255, 0, "Green - Elevator Zeroed")
+              .colorSet(0, 170, 0, "Green - Elevator Zeroed")
+              .withTimeout(2)
+              .andThen(s.elevatorLEDSubsystem.colorSet(0, 0, 0, "LED off"))
               .ignoringDisable(true));
-      hasBeenZeroed.onFalse(
-          s.elevatorLEDSubsystem
-              .colorSet(255, 0, 0, "Red - Elevator Not Zeroed")
-              .ignoringDisable(false));
+      RobotModeTriggers.disabled()
+          .and(hasBeenZeroed.negate())
+          .onTrue(
+              s.elevatorLEDSubsystem
+                  .blink(120, 0, 0, "Red - Elevator Not Zeroed")
+                  .ignoringDisable(true));
     }
     RobotModeTriggers.autonomous()
         .whileTrue(s.elevatorLEDSubsystem.animate(LEDPattern.rainbow(255, 255), "Auto Rainbow"));
+    s.elevatorLEDSubsystem
+        .thirtySecondsLeft()
+        .onTrue(
+            Commands.sequence(
+                s.elevatorLEDSubsystem.blinky(255, 0, 0, "red half blink - 30 sec remaining"),
+                s.elevatorLEDSubsystem.blinky(
+                    255, 255, 0, "Yellow half blink - 30 sec remaining")));
```

Comment:
Right now, this won't work since each `blinky()` command will repeat on-off for its color forever. Instead, we can do a red-yellow blink by running solid red (using `colorSet()`) with a timeout, then running solid yellow (using `colorSet()`) with a timeout, all in a repeating sequence. (This means that we can remove the `blinky()` factory method)